dev/ToastNotifications/ToastNotificationUtility.h

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+#include "pch.h"
+#include <winrt/Windows.Foundation.h>
+#include <ToastActivatedEventArgs.h>
+
+const std::wstring appIdentifierPath = L"Software\\Classes\\AppUserModelId\\";
```

Comment:
Nit, var naming:

```suggestion
const std::wstring c_appIdentifierPath = L"Software\\Classes\\AppUserModelId\\";
```

Same for the var below.